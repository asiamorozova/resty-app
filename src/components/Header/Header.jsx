import React from 'react';
import styles from './header.css';

const Header = ({ style }) => (
  <h3 style={style}>Bootleg Postman</h3>
);

export default Header;
