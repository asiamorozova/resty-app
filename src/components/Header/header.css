.h1{
   box-shadow: yellowgreen;
   padding-left: 1rem;
}