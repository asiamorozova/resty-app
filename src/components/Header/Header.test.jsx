import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

describe('App component', () => {
  it('renders Header and Resty', () => {
    const wrapper = shallow(
      <Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
