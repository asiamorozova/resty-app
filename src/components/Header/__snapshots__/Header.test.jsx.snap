// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders Header and Resty 1`] = `
<h3>
  Bootleg Postman
</h3>
`;
